db/migrate/20200623170047_add_salesforce_id_to_school.rb

```diff
@@ -0,0 +1,5 @@
+class AddSalesforceIdToSchool < ActiveRecord::Migration[6.0]
+  def change
+    add_column :schools, :salesforce_id, :string
```

Comment:
ditto above, just modify the one migration that creates the table. 

is this ID globally unique?  If so, add a database constraint to ensure that we don't get duplicates.  Can also add a validation in the model to ensure it is unique (the one in the model gives nicer error messages, the one in the database is still required to prevent race condition issues, let me know if you need clarification here)

Reply:
done - added model and database validation to that field (I read that was best practice)
this should be both unique and required